README_sdl.txt

```diff
@@ -119,4 +109,5 @@ easiest to start a local webserver, and then navigate to the examples using a
 web browser. E.g. you could use the Python's web server module which prints out
 a URL you can open:
 
+cd examples
```

Comment:
Let's leave this off as well, as running the server in the build directory lets you play the demos as well.